examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>it should be <code>merlin-tensorflow</code> container. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>corrected the text and the url</p>